<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>801</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="TabMain">
     <attribute name="title">
      <string>API</string>
     </attribute>
     <widget class="QWidget" name="centralwidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>-720</x>
        <y>-20</y>
        <width>805</width>
        <height>611</height>
       </rect>
      </property>
     </widget>
     <widget class="QFrame" name="API">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>551</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="Quit">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>10</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
      <widget class="QFrame" name="Gestionnaire">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>151</width>
         <height>101</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="GestionnaireL">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>        State manager</string>
        </property>
       </widget>
       <widget class="QFrame" name="BarDB">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="EtatDBL">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>0</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>DataBase</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="BarRequete">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="EtatRequeteL">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>          Request</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="RegistreL">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QLabel" name="IndicationRegistre">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>16</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Log">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>70</y>
          <width>41</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Log</string>
        </property>
       </widget>
      </widget>
      <widget class="QComboBox" name="Menu">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Choise selection</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Affichage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interaction</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Autonomie</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="Selection">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="RequestLine">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>381</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Drop">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>50</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Drop</string>
       </property>
      </widget>
      <widget class="QTableView" name="AfficheSQL">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>160</y>
         <width>491</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableView" name="ListeTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>191</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="CompleteInfo">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>381</width>
         <height>91</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="Topic">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>381</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="TableSelectionner">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>100</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Table selection : </string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="TabLog">
     <attribute name="title">
      <string>LOGS</string>
     </attribute>
     <widget class="QTabWidget" name="MenuLog">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>661</width>
        <height>431</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="SQL">
       <attribute name="title">
        <string>Log SQL</string>
       </attribute>
       <widget class="QTextEdit" name="LogSQL">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>631</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="DataBase">
       <attribute name="title">
        <string>Log DataBase</string>
       </attribute>
       <widget class="QTextEdit" name="LogDB">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>631</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="Clear">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>470</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Sauvegarder">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>470</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
